src/Security/Authentication/JwtBearer/src/JwtBearerEvents.cs

```diff
@@ -15,6 +15,10 @@ public class JwtBearerEvents
         /// Invoked if exceptions are thrown during request processing. The exceptions will be re-thrown after this event unless suppressed.
         /// </summary>
         public Func<AuthenticationFailedContext, Task> OnAuthenticationFailed { get; set; } = context => Task.CompletedTask;
+        /// <summary>
+        /// Invoked if exceptions are thrown during the authorzation process.  The exceptions will be re-thrown after this event unless supressed.  
```

Comment:
This text is wrong. 403's aren't the result of exceptions.

Reply:
Yes thank you.  I have updated this to follow the pattern for the OnChallenge